### ISSUE TITLE
feat: Allow external (absolute) file paths

### ISSUE THREAD

**Issue by phillipmohr** [samk-dev/nuxt-translation-manager#22]
It would be great if the `langDir` would also allow absolute paths. 

I'd like to host my CSV on Supabase so people in my team can easily edit it without the need to build a UI in my application.

**Comment by samk-dev** [comment]
Hey thank you Philip. Do you mean to host the csv file on Supabase storage? Please could you past here an example of how would the url look like?

For the time being the teams I collaborate with the edit the csv in `google docs` or any CSV editor they have and they send me the file and I just drop it in the langDir

**Comment by phillipmohr** [comment]
Yes, it would be nice if the CSV can be hosted basically anywhere. 

The URL in Supabase can look like this: 
https://mysupabaseproject.supabase.co/storage/v1/object/public/myfolder/anotherfolder/translations.csv

In the meantime I have just build the UI for the CSV upload but I believe it could become handy if the CSV could also be hosted somewhere else

**Comment by phillipmohr** [comment]
Hi, I just wanted to add something: I'm hosting on Vercel and it seems like Vercel doesn't allow writing files, meaning this doesn't work
```js
const filePath = path.join(process.cwd(), 'public', 'locales', 'translations.csv');
fs.writeFileSync(filePath, csvFile.data);
```

It is recommended to use an external provider but unfortunately I can't specify an absolute external URL. 

So for me, this is unfortunately not a "nice to have" but a must have because otherwise I can't use the nuxt translation manager on my prod environment (Vercel)
